app/resources/home.html.template

```diff
@@ -1,13 +1,83 @@
-{# Generic, default home page for a new site.  Fill in content below.       #}
-
 {% extends "static-base.html.template" %}
+{% load i18n %}
 
 {% block content %}
-  <h1>Welcome</h1>
 
-  <p>You are now running Person Finder.
+  {% if config.brand == "google" %}
+
+    <h1>
+      Google Person Finder helps people reconnect with friends and loved ones
+      in the aftermath of natural and humanitarian disasters.
```

Comment:
The home page has been supporting only English and Japanese. Japanese version is in a separated file home.html.template:ja. It would be an option to add "trans" tags to i18n this file, but I first want to simply move the resource files as is from the dynamic bundle to Github.

We need discussion before deciding to i18n this file because it would cost some money.